src/main/java/it/pagopa/pn/deliverypush/action2/DigitalWorkFlowHandler.java

```diff
@@ -207,26 +213,37 @@ public void handleExternalChannelResponse(ExtChannelResponse response, TimelineE
 
                     nextWorkFlowAction(notification, recIndex, lastAttemptMade);
```

Comment:
NB: dopo l'ultima modifica relativa a publicregistry, il "prossimo step" andrà sempre in exception ovviamente.
immagino sia voluto, ma volevo cmq farlo presente